src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -1263,8 +1263,24 @@ class misc_commandscript : public CommandScript
         // Subtract
         if (count < 0)
         {
-            playerTarget->DestroyItemCount(itemId, -count, true, false);
-            handler->PSendSysMessage(LANG_REMOVEITEM, itemId, -count, handler->GetNameLink(playerTarget).c_str());
+            uint32 destroyedItemCount = playerTarget->DestroyItemCount(itemId, -count, true, false);
+            
+            if (destroyedItemCount > 0) {
```

Comment:
Surprised noone mentioned this yet: codestyle, placement of opening brace { in all of your changes must be on new line

Reply:
gotcha. i should have noticed that.